topdown/http.go

```diff
@@ -1188,23 +1195,19 @@ func newInterQueryCache(bctx BuiltinContext, key ast.Object, forceCacheParams *f
 func (c *interQueryCache) CheckCache() (ast.Value, error) {
 	var err error
 
-	c.httpReq, c.httpClient, err = createHTTPRequest(c.bctx, c.key)
-	if err != nil {
-		return nil, handleHTTPSendErr(c.bctx, err)
-	}
-
 	c.forceJSONDecode, err = getBoolValFromReqObj(c.key, ast.StringTerm("force_json_decode"))
 	if err != nil {
 		return nil, handleHTTPSendErr(c.bctx, err)
 	}
 
-	resp, err := checkHTTPSendInterQueryCache(c.bctx, c.key, c.httpReq, c.httpClient, c.forceJSONDecode, c.forceCacheParams)
+	resp, err := c.checkHTTPSendInterQueryCache(c.bctx, c.key, c.forceJSONDecode, c.forceCacheParams)
```

Comment:
🤔 Hrmm if this is method now has `c` as a receiver, what's the reason to pass `c.bctx`, `c.key`, etc?

Reply:
Updated. Removed the args to `c.checkHTTPSendInterQueryCache`.